src/cocaine/burlak/config.py

```diff
@@ -101,6 +108,21 @@ def make_sharding_config(d):
     )
 
 
+def make_netlink_config(d):
+    Netlink = namedtuple('Netlink', [
+        'default_name',
+        'speed_mbits',
+    ])
+
+    default_name = d.get('default_name', Defaults.NETLINK_SPEED_MBITS)
+    speed_mbits = d.get('speed_mbits', Defaults.NETLINK_NAME)
```

Comment:
`Defaults.NETLINK_NAME` is used for `speed_mbits` and `Defaults.NETLINK_SPEED_MBITS` for `default_name`